### PR TITLE
Update plannertask-update.md

### PR DESCRIPTION
**Pull request by SOURSH**
This change is part of "Project Planner Integration" work. Roadmap id - https://www.microsoft.com/en-us/microsoft-365/roadmap?rtc=1&filters=Planner&searchterms=382207

### CODE REVIEW

File: api-reference/v1.0/api/plannertask-update.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ PATCH /planner/tasks/{id}
 |startDateTime|DateTimeOffset|Date and time at which the task starts. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.|
 |title|String|Title of the task.|
 
+> [!NOTE]
+> For Project tasks that are replicated in Planner, only the **percentComplete** field can be updated.
```
Comment (Lauragra) [inline api-reference/v1.0/api/plannertask-update.md:58]: @SOURSH , please apply this change to the plannertask-update file in the /beta folder as well.

**Comment by SOURSH** [comment]
@microsoft-github-policy-service agree company="Microsoft"


Regards,
Sourabh
________________________________

File: api-reference/v1.0/api/plannertask-update.md (hunk 1 above)
Comment (SOURSH) [reply api-reference/v1.0/api/plannertask-update.md]: @Lauragra I have made the suggested changes. Please review

**Comment by SOURSH** [comment]
@Lauragra I don't have write access, I have incorporated the suggestions, can you please merge this? tx!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**